text/0000-customisability.md

```diff
@@ -0,0 +1,80 @@
+- Start Date: 2016-10-25
+- RFC Issue: #3
+- RFC PR: 
+- Implementation Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Customisation of Rustfmt should be allowed (via a `rustfmt.toml` file), but not
+encouraged.
+
+
+# Details
+[details]: #details
+
+Users may create a `rustfmt.toml` or `.rustfmt.toml` in their project directory
```

Comment:
The `C` in `Cargo.toml` was, apparently, chosen so it'd be sorted near other build system files (`Makefile`).

`rustfmt` is not a build system so IMO does not need a capital character.